packages/backend/src/server/api/endpoints/users.ts

```diff
@@ -70,14 +72,29 @@ export default class extends Endpoint<typeof meta, typeof paramDef> { // eslint-
 			if (ps.hostname) {
 				query.andWhere('user.host = :hostname', { hostname: ps.hostname.toLowerCase() });
 			}
-
+			const chartUsers: { userId: string; count: number; }[] = [];
+			if (ps.sort?.endsWith('pv')) {
+				await this.perUserPvChart.getChartUsers('hour', 0, null, ps.limit, ps.offset).then(users => {
```

Comment:
```suggestion
				await this.perUserPvChart.getChartUsers('day', 0, null, ps.limit, ps.offset).then(users => {
```
ioだからいいけど一般サーバーではシビアすぎるかもしれない